File: test_login.js
const url = "http://localhost:54321/auth/v1/token?grant_type=password";
const headers = {
    "apikey": "sb_publishable_ACJWlzQHlZjBrEguHvfOxg_3BJgxAaH",
    "Content-Type": "application/json"
};
const data = {
    "email": "admin@kapeuno.com",
    "password": "password"
};

fetch(url, {
    method: "POST",
    headers: headers,
    body: JSON.stringify(data)
})
.then(response => {
    console.log(`Status Code: ${response.status}`);
    return response.text().then(text => ({status: response.status, text}));
})
.then(({status, text}) => {
    if (status !== 200) {
        console.log(text);
    } else {
        console.log("Login successful.");
        const resp_data = JSON.parse(text);
        console.log("Access Token length:", resp_data.access_token ? resp_data.access_token.length : 0);
    }
})
.catch(e => console.log("Request failed:", e));

File: test_login.py
import requests
import json

url = "http://localhost:54321/auth/v1/token?grant_type=password"
headers = {
    "apikey": "sb_publishable_ACJWlzQHlZjBrEguHvfOxg_3BJgxAaH",
    "Content-Type": "application/json"
}
data = {
    "email": "admin@kapeuno.com",
    "password": "password"
}

try:
    response = requests.post(url, headers=headers, json=data)
    print(f"Status Code: {response.status_code}")
    if response.status_code != 200:
        print(response.text)
    else:
        print("Login successful.")
        resp_data = response.json()
        print("Access Token length:", len(resp_data.get("access_token", "")))
except Exception as e:
    print("Request failed:", e)
